packages/opentelemetry-node/docs/getting-started.mdx

```diff
@@ -1,19 +1,8 @@
-# Introduction
-
-The Elastic OpenTelemetry Distribution for Node.js (the "Distro") is a Node.js
-package that provides:
-
-- an easy way to instrument your application with OpenTelemetry, and
-- configuration defaults for best usage.
-
-Use the Distro to start the OpenTelemetry SDK with your Node.js application to automatically
-capture tracing data, performance metrics, and logs. Traces, metrics, and logs are sent
-to any OTLP collector you choose. Use an [Elastic Observability](https://www.elastic.co/observability)
-deployment -- hosted in Elastic's cloud or on-premises -- to monitor your applications, create alerts,
-and quickly identify root causes of service issues.
-
-
-# Getting Started
+---
+id: otelNodeGetStarted
+slug: /otel-nodejs/get-started
+title: Get started
+---
 
 This getting started guide will show how to use this Distro to instrument your Node.js application and send OpenTelemetry data to an Elastic Observability deployment. Note, however, that as an OpenTelemetry SDK, it supports sending data to any OTLP endpoint, e.g. an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/).
```

Comment:
Given that the above section is dropped now, I think it would be good to change the first use of "Distro" here to defined it.

```suggestion
This getting started guide will show how to use Elastic OpenTelemetry Distribution for Node.js (the "Distro") to instrument your Node.js application and send OpenTelemetry data to an Elastic Observability deployment. Note, however, that as an OpenTelemetry SDK, it supports sending data to any OTLP endpoint, e.g. an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/).
```

Reply:
Good call!